src/middlewared/middlewared/etc_files/cron.py

```diff
@@ -0,0 +1,5 @@
+import shutil
+
+
+def render(service, middleware):
+    shutil.copyfile('/conf/base/etc/crontab.template', '/etc/crontab')
```

Comment:
We can just move this file to template folder and avoid doing this

Reply:
Ah, of course